pkg/grub/rootfs.cfg

```diff
@@ -362,7 +362,10 @@ function set_kubevirt_boot {
 }
 
 function set_eve_flavor {
-   cat -s eve_flavor /etc/eve-hv-type
+   # don't overwrite as the XEN installer still uses KVM (and sets it explicitly)
+   if [ "$eve_flavor" = "" ] ; then
+       cat -s eve_flavor /etc/eve-hv-type
```

Comment:
@christoph-zededa, I tested these changes and it works. However, `/etc/eve-hy-type` will still contain the xen string, is that a problem?

Reply:
@rene no, this is intended. We just run KVM installer for XEN build